verl/trainer/fsdp_sft_trainer.py

```diff
@@ -439,6 +439,64 @@ def save_checkpoint(self, step):
             if self.config.trainer.default_hdfs_dir:
                 hdfs_io.makedirs(self.config.trainer.default_hdfs_dir, exist_ok=True)
                 hdfs_io.copy(src=path, dst=self.config.trainer.default_hdfs_dir, dirs_exist_ok=True)
+
+            # Push to hub if enabled
+            if self.config.trainer.push_to_hub:
```

Comment:
thanks for the contribution! 
could you consolidate these in a util function so that the trainer code looks less bulky? thanks!

Reply:
Sure, added a utility function